exercises/bracket-push/stack.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-8.6
```

Comment:
same q as https://github.com/exercism/xhaskell/pull/511#discussion_r109295114 - same across all exercises, or allow to be different?